(Docs): Added 24pullRequests announcement post (#158)
uptane/uptane.github.io: blog/2023-12-01-24pullRequests/index.md

---
title: "Uptane Joins 24 Pull Requests for a Season of Open Source Giving ❄️"
authors:
  - Abhij
---

![](bannerImage.png)

> As the winter season approaches, [Uptane](https://uptane.org/) is thrilled to announce its participation in the annual 24 Pull Requests event! This initiative is a testament to our commitment to collaboration and community building.

Reviewer: ```suggestion
As winter approaches, [Uptane](https://uptane.org/) is thrilled to announce its participation in the annual 24 Pull Requests event! This initiative is a testament to our  project's continuing commitment to collaboration and community building.

```

## 🤝 About Uptane:

> Uptane stands as a robust framework designed to fortify the security of software updates within the automotive industry. Uptane addresses the need for a reliable solution, ensuring the safety and integrity of software updates for vehicles.

Reviewer: ```suggestion
Uptane is a robust framework designed to fortify the security of software updates within the automotive industry. It  addresses the growing need for a reliable solution that can ensure the safety and integrity of software updates for vehicles.
```

## 🎁 24 Pull Requests:

> The [24 Pull Requests](https://24pullrequests.com/) event is an annual community-organized celebration, inviting developers to contribute to open source projects during the holiday season. Join us in spreading the joy of collaboration by submitting 24 pull requests between December 1 and December 24.

Reviewer: ```suggestion
The [24 Pull Requests](https://24pullrequests.com/) event is an annual community-organized celebration in which developers are invited to contribute to open source projects over the holiday season. Join us in spreading the joy of collaboration by submitting 24 pull requests between December 1 and December 24.
```
